feature/qos/ate_tests/qos_basic_test/qos_basic_test.go

```diff
@@ -397,23 +397,15 @@ func TestBasicConfigWithTraffic(t *testing.T) {
 
 			counters := make(map[string]map[string]uint64)
 			var counterNames []string
-			if !deviations.QOSDroppedOctets(dut) {
-				counterNames = []string{
 
-					"ateOutPkts", "ateInPkts", "dutQosPktsBeforeTraffic", "dutQosOctetsBeforeTraffic",
-					"dutQosPktsAfterTraffic", "dutQosOctetsAfterTraffic", "dutQosDroppedPktsBeforeTraffic",
-					"dutQosDroppedOctetsBeforeTraffic", "dutQosDroppedPktsAfterTraffic",
-					"dutQosDroppedOctetsAfterTraffic",
-				}
-			} else {
-				counterNames = []string{
-
-					"ateOutPkts", "ateInPkts", "dutQosPktsBeforeTraffic", "dutQosOctetsBeforeTraffic",
-					"dutQosPktsAfterTraffic", "dutQosOctetsAfterTraffic", "dutQosDroppedPktsBeforeTraffic",
-					"dutQosDroppedPktsAfterTraffic",
-				}
+			counterNames = []string{
```

Comment:
Fix static check failure - `should merge variable declaration with assignment on next line `

Reply:
Fixed it